app/js/arethusa.core/services/navigator.js

```diff
@@ -95,6 +95,9 @@ angular.module('arethusa.core').service('navigator', [
     function currentIds () {
       return arethusaUtil.map(currentSentenceObjs(), 'id');
     }
+    this.currentSubdocs = function() {
+      return arethusaUtil.map(currentSentenceObjs(), 'subdoc');
```

Comment:
Do you know if there's any reason not to return the first element of this array? In the treebanks I tested this with there was never more (or less) than one element in this array.

Reply:
The subdoc attribute is optional. It is always used on treebank files that are created through the Perseids workflow (I think), but it's possible that someone could upload a treebank file that doesn't use that field. But in that case you would just get an empty array.  There should never been more than one subdoc per sentence using the current treebank schema.

Reply:
Great, I'll update this to `arethusaUtil.map(currentSentenceObjs(), 'subdoc')[0]`. That should make the API easier.